src/main/java/mate/academy/onlinebookstore/entity/Role.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.onlinebookstore.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.springframework.security.core.GrantedAuthority;
+
+@Entity
+@Data
+@Table(name = "roles")
+public class Role implements GrantedAuthority {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false, unique = true)
+    @Enumerated(EnumType.STRING)
+    private RoleName name;
+
+    @Override
+    public String getAuthority() {
+        return "ROLE_" + name.name();
```

Comment:
make "ROLE_" constant, for performance\memory improvements 
```suggestion
        return "ROLE_" + name.name();
```